{"name": "test_user_should_see_empty_bucket", "status": "skipped", "statusDetails": {"message": "XFAIL -- fixing this bug right now. --\n\nAssertionError:  -- Success remove message not presented", "trace": "E   AssertionError:  -- Success remove message not presented"}, "attachments": [{"name": "stdout", "source": "56c87749-86b0-410e-a46c-8d6fa577feba-attachment.txt", "type": "text/plain"}], "start": 1608537862695, "stop": 1608537872833, "uuid": "e9b4482a-f892-43e4-8bb4-0516c7e66a0a", "historyId": "c4aa36e359cba9c046ae8bd6b03ab344", "testCaseId": "7f0bef2cb364ece1f150bc86314501db", "fullName": "test_product_page.TestUserProductPage#test_user_should_see_empty_bucket", "labels": [{"name": "tag", "value": "User"}, {"name": "tag", "value": "@pytest.mark.xfail(reason='-- fixing this bug right now. --')"}, {"name": "suite", "value": "test_product_page"}, {"name": "subSuite", "value": "TestUserProductPage"}, {"name": "host", "value": "Netbook"}, {"name": "thread", "value": "14872-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_product_page"}]}